doc/api/tty.md

```diff
@@ -82,12 +82,15 @@ or `writeStream.rows` properties have changed. No arguments are passed to the
 listener callback when called.
 
 ```js
+process.stdin.setRawMode(true);
 process.stdout.on('resize', () => {
   console.log('screen size has changed!');
   console.log(`${process.stdout.columns}x${process.stdout.rows}`);
 });
 ```
 
+*Note:* Unrealiable event handler execution on all Windows platforms.
```

Comment:
1. `Unrealiable` → `Unreliable`
2. What exactly do you want to say? Please try to use whole sentences if possible. Do you mean that the execution of this event is unreliable on Windows?